resources/smart-contracts/glp-compounder.md

```diff
@@ -5,4 +5,10 @@
 * Address: [https://arbiscan.io/address/0x9363e5e4a7aDfB346BEA0fe87a8BD02fddA48855](https://arbiscan.io/address/0x9363e5e4a7aDfB346BEA0fe87a8BD02fddA48855)
 * Vault Asset: GLP
 * Vault Shares: fortGLP
-* Strategy: GLP is being staked in GMX, earning more GLP from GMX traders and swappers.
+* Strategy: GLP is being staked in GMX, earning more esGMX and multiplier points
+  from GMX traders and swappers. The earned esGMX is never vested but instead
```

Comment:
"GLP is being staked in GMX, earning fees from traders and token incentives from GMX."

Reply:
other than that lgtm.